src/components/accession2/create/Accession2Create.tsx

```diff
@@ -44,8 +43,40 @@ export default function CreateAccession(): JSX.Element {
   const history = useHistory();
   const snackbar = useSnackbar();
   const [validateFields, setValidateFields] = useState<boolean>(false);
+  const [selectedSeedBank, setSelectedSeedBank] = useState<Facility>();
+  const timeZoneFeatureEnabled = isEnabled('Timezones');
+  const tz = useLocationTimeZone().get(timeZoneFeatureEnabled ? selectedSeedBank : undefined);
+  const [timeZone, setTimeZone] = useState<string>(tz.id);
+  const { selectedOrganization } = useOrganization();
+
+  const defaultAccession = (): AccessionPostRequestBody =>
+    ({
+      state: 'Awaiting Check-In',
+      receivedDate: getTodaysDateFormatted(timeZone),
+    } as AccessionPostRequestBody);
+
   const [record, setRecord, onChange] = useForm<AccessionPostRequestBody>(defaultAccession());
 
+  useEffect(() => {
+    if (record.facilityId) {
+      const accessionSeedBank = getSeedBank(selectedOrganization, record.facilityId);
+      setSelectedSeedBank(accessionSeedBank);
+    }
+  }, [record.facilityId, selectedOrganization]);
+
+  useEffect(() => {
+    setTimeZone(tz.id);
+  }, [tz]);
```

Comment:
@constanzauanini can we do a test to make sure this isn't triggering a lot because tz is not in the state? maybe a console.log will show

Reply:
I see it just one time